litmus-portal/frontend/src/components/YamlEditor/Editor.tsx

```diff
@@ -118,7 +118,7 @@ const YamlEditor: React.FC<YamlEditorProps> = ({
         'ace_gutter-cell'
       );
       for (let i = 0; i < nodeStyleErrorList.length; i += 1) {
-        (nodeStyleErrorList[i] as any).style.backgroundColor = '#1C1C1C';
```

Comment:
Kindly import colors from theme file if possible instead of hardcoded colors

Reply:
These changes will be with v2 redesigning.

Reply:
If we are making any change in the Hex then it can be from litmus theme itself now also as hard coded colors do look good.

Reply:
There are two different colours in the editor right now, latest code is available on v2 branch

Reply:
This change makes the colour uniform as was in the design before some breaking changes.